josh.cljs

```diff
@@ -486,16 +499,16 @@
         files ["index.html" "main.cljs" "style.css"]]
     (js/console.log "Copying example files here.")
     (p/do!
-      (->> files
```

Comment:
My editor disagrees with these indent changes. Is it possible to submit a PR without all the whitespace-only and linebreak changes? Or issue that as a separate PR?